tests/Browser/BlackoutsPageTest.php

```diff
@@ -28,6 +28,36 @@ public function setUp(): void
         $seeder->run();
         User::first()->assignRole('super-admin');
     }
+    public function testCanCreateBlackouts()
+    {
+        Room::factory()->create();
+
+        AcademicDate::create([
+            'semester' => 'Fall',
+            'start_date' => Carbon::now()->subDays(2),
+            'end_date' => Carbon::now()->addDays(5)
+        ]);
+
+        $this->browse(function (Browser $browser) {
+            $browser->loginAs(User::first())->visit('/admin/rooms/1/blackouts');
+
+            $browser->within(new DateTimePicker("start"), function ($browser) {
+                $browser->setDatetime(15, 7);
+            })->pause(1000);
+
+            $browser->within(new DateTimePicker("end"), function ($browser) {
+                $browser->setDatetime(15, 8);
+            })->pause(1000);
+
+            $browser->type('#name', 'alex')
+                ->press('#submit')
+                ->pause(8000);
+
+                $browser->assertSee('alex');
```

Comment:
we can probably cut down on the pause by waiting for a text change.

Reply:
Does that just replace the pause and assert, or just the pause?

You do have a point though, the role page looks very similar so I could do what was done there,